### ISSUE TITLE
Similar name is provided for "Icons" Present: A11y_.NET 9 template_.NET 9 template_Project_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26185]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG4.1.2; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Name Role Value;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till project section, and press enter on any project, Project screen will open, 
4. Tab till icons.
5. Observe that No name is provided for "checkboxes", "clear" and "New Task" button.

### Actual Result:
Similar name is provided for "Icons" Present

### Expected Result:
Proper and appropriate name should be provided.

### User Impact:
Screen reader users will get confused if similar name is provided for all the controls.

### Attachment:
https://github.com/user-attachments/assets/0d868f14-e1b0-413e-866b-f62261b7b057

**Comment by AnKushSingh05** [comment]
Issue is fixed, Hence closing the bug

![Image](https://github.com/user-attachments/assets/b8ab4d8d-c3cf-4f97-a587-b7891baf5de9)

![Image](https://github.com/user-attachments/assets/b72f2844-4970-47b2-b802-2fe5e8ab02d9)

![Image](https://github.com/user-attachments/assets/2a87e73e-e167-4bc1-a5db-d6d777e05c17)
